app/jobs/subscription_confirm_job.rb

```diff
@@ -84,7 +84,9 @@ def setup_payment!(order)
   def authorize_payment!(order)
     return if order.subscription.payment_method.class != Spree::Gateway::StripeSCA
 
-    OrderManagement::Subscriptions::StripeScaPaymentAuthorize.new(order).call!
+    OrderManagement::Order::StripeScaPaymentAuthorize.new(order).
+      extend(OrderManagement::Order::SendAuthorizationEmails).
```

Comment:
nice! I didn't think about this possibility :clap: I had [SimpleDelegator](https://ruby-doc.org/stdlib-2.4.1/libdoc/delegate/rdoc/SimpleDelegator.html) in mind but this is good too.